// react-native-screens/lib/commonjs/fabric/ScreenStackHeaderSubview.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var _react = _interopRequireDefault(require("react"));

var _ScreenStackHeaderSubviewNativeComponent = _interopRequireDefault(require("./ScreenStackHeaderSubviewNativeComponent"));

var _reactNative = require("react-native");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

const styles = _reactNative.StyleSheet.create({
  headerSubview: {
    position: 'absolute',
    top: 0,
    right: 0,
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'center'
  }
});

function ScreenStackHeaderSubview(props) {
  return /*#__PURE__*/_react.default.createElement(_ScreenStackHeaderSubviewNativeComponent.default, _extends({}, props, {
    style: styles.headerSubview
  }));
}

var _default = ScreenStackHeaderSubview;
exports.default = _default;
